Tests/StoreKitTests/SKProductTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SKProductTests.swift
+//  SwifterSwift
+//
+//  Created by Batuhan Saka on 26.12.2019.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(StoreKit)
+import StoreKit
+
+public extension SKProduct {
+
```

Comment:
Please make this extension `private`

Reply:
It was already fileprivate in the beginning. I changed because of Error #1. But, there had a problem while the tests running via TravisCI. But, I did not face any problem during the running tests in Xcode.

I will give it a try.

Thanks for the review.

Reply:
Requested changes have been applied. @omaralbeik